api/v1alpha1/account_types.go

```diff
@@ -35,23 +35,61 @@ type AccountSpec struct {
 	// +optional
 	ClaimLink string `json:"claimLink"`
 	// +optional
-	ClaimLinkNamespace string      `json:"claimLinkNamespace,omitempty"`
-	LegalEntity        LegalEntity `json:"legalEntity,omitempty"`
-	ManualSTSMode      bool        `json:"manualSTSMode,omitempty"`
-	AccountPool        string      `json:"accountPool,omitempty"`
+	ClaimLinkNamespace    string                `json:"claimLinkNamespace,omitempty"`
+	LegalEntity           LegalEntity           `json:"legalEntity,omitempty"`
+	ManualSTSMode         bool                  `json:"manualSTSMode,omitempty"`
+	AccountPool           string                `json:"accountPool,omitempty"`
+	RegionalServiceQuotas RegionalServiceQuotas `json:"regionalServiceQuotas,omitempty"`
 }
 
+type RegionalServiceQuotas map[string]AccountServiceQuota
+
+// +k8s:openapi-gen=true
+type AccountServiceQuota map[SupportedServiceQuotas]*ServiceQuotaStatus
+
+type ServiceQuotaStatus struct {
+	Value  int                  `json:"value"`
+	Status ServiceRequestStatus `json:"status"`
+}
+
+type ServiceRequestStatus string
+
+const (
+	ServiceRequestTodo       ServiceRequestStatus = "TODO"
+	ServiceRequestInProgress ServiceRequestStatus = "IN_PROGRESS"
+	ServiceRequestCompleted  ServiceRequestStatus = "COMPLETED"
+	ServiceRequestDenied     ServiceRequestStatus = "DENIED"
+)
+
+type SupportedServiceQuotas string
```

Comment:
It seems odd to create custom aliases for primitive types, but I think it makes sense in this case for clarity.

Reply:
It helps to limit the values it can take at least - kind of like a poor man's enum.